packages/components/inputs/localized-money-input/src/localized-money-input.tsx

```diff
@@ -18,23 +24,203 @@ import {
+   */
+  selectedCurrency: string;
+  /**
+   * Called when input is blurred
+   */
+  onBlur?: (event: TEvent) => void;
+  /**
+   * Called when input is focused
+   */
+  onFocus?: FocusEventHandler;
+  /**
+   * Will hide the currency expansion controls when set to `true`. All currencies will be shown when set to `true`.
+   */
+  hideCurrencyExpansionControls?: boolean;
+  /**
+   * Controls whether one or all currencirs are visible by default
+   */
+  defaultExpandCurrencies?: boolean;
+  /**
+   * Indicates that the input cannot be modified (e.g not authorized, or changes currently saving).
+   */
+  isDisabled?: boolean;
+  /**
+   * Indicates that the field is displaying read-only content
+   */
+  isReadOnly?: boolean;
+  /**
+   * Placeholder text for the input
+   */
+  placeholder?: Record<string, string>;
+  /**
+   * Horizontal size limit of the input fields.
+   */
+  horizontalConstraint?:
+    | 3
+    | 4
+    | 5
+    | 6
+    | 7
+    | 8
+    | 9
+    | 10
+    | 11
+    | 12
+    | 13
+    | 14
+    | 15
+    | 16
+    | 'scale'
+    | 'auto';
+  /**
+   * Indicates that input has errors
+   */
+  hasError?: boolean;
+  /**
+   * Control to indicate on the input if there are values that are potentially invalid
+   */
+  hasWarning?: boolean;
+  /**
+   * A map of errors. Error messages for known errors are rendered automatically.
+   * <br>
+   * Unknown errors will be forwarded to `renderError`
+   */
+  errors?: Record<string, ReactNode>;
+  /**
+   * A map of warnings.
+   */
+  warnings?: Record<string, ReactNode>;
+};
+
+type TLocalizedInputProps = {
+  /**
+   * Used as HTML id property.
+   */
+  id?: string;
+  /**
+   * The prefix used to create a HTML `name` property for the amount input field (`${name}.amount`).
+   */
+  name?: string;
+  /**
+   * value of possible currency
+   */
+  value: TValue;
+  /**
+   * Called with the event of the input.
+   */
+  onChange?: ChangeEventHandler;
+  /**
+   * Called with the event of the input.
+   */
+  currency: string;
+  /**
+   * Called when input is blurred
+   */
+  onBlur?: (event: TEvent) => void;
+  /**
+   * Called when input is focused
+   */
+  onFocus?: FocusEventHandler;
+  /**
+   * Indicates that the input cannot be modified (e.g not authorized, or changes currently saving).
+   */
+  isDisabled?: boolean;
+  /**
+   * Indicates that the field is displaying read-only content
+   */
+  isReadOnly?: boolean;
+  /**
+   * Indicates that input has errors
+   */
+  hasError?: boolean;
+  /**
+   * Control to indicate on the input if there are values that are potentially invalid
+   */
+  hasWarning?: boolean;
+  /**
+   * Placeholder text for the input
+   */
+  placeholder?: string;
+  /**
+   * A map of errors. Error messages for known errors are rendered automatically.
+   * <br>
+   * Unknown errors will be forwarded to `renderError`
+   */
+  error?: ReactNode;
+  /**
+   * HTML node to display warning
+   */
+  warning?: ReactNode;
+  intl?: IntlShape;
```

Comment:
Is this used?

Reply:
Yes. used in the code [here](https://github.com/commercetools/ui-kit/blob/19f3000d64872acc5c03dd59e568b84eb66961e8/packages/components/inputs/localized-money-input/src/localized-money-input.tsx#L357) so I had to add it

Reply:
Yes but that's not a prop. It's defined using `useIntl`

https://github.com/commercetools/ui-kit/blob/19f3000d64872acc5c03dd59e568b84eb66961e8/packages/components/inputs/localized-money-input/src/localized-money-input.tsx#L277

Reply:
Oh, thats true.